### ISSUE TITLE
Top level characters in the into the light game not having a legacy path 

### ISSUE THREAD

**Issue by kernowboy** [Dugy/Legend_of_the_Invincibles#87]
Top  characters in the into the light game not having a legacy path.

So far this applies to:

Level 5
Predator
Mystical Elvish deadeye
Elvish Paragon

Level 4
Woodland enchantress
Auxiliary master
Epic saber 

Playing with following details
The Battle for Wesnoth version 1.14.2
Running on Darwin 17.3.0 Darwin Kernel Version 17.3.0: Mon Nov  6 21:19:17 PST 2017; root:xnu-4570.32.1~1/RELEASE_ARM64_T8011 iPad7,1

Game paths
==========

Data dir:        /private/var/containers/Bundle/Application/F0682B9D-A9F1-47A9-8D35-2289A2F903ED/battleforwesnoth-iPad.app
User config dir: /private/var/USER/Containers/Data/Application/235F83BD-513E-413C-996C-91564162FE79/Library/Application Support/wesnoth.org/iWesnoth/.wesnoth1.14
User data dir:   /private/var/USER/Containers/Data/Application/235F83BD-513E-413C-996C-91564162FE79/Library/Application Support/wesnoth.org/iWesnoth/.wesnoth1.14
Saves dir:       /private/var/USER/Containers/Data/Application/235F83BD-513E-413C-996C-91564162FE79/Library/Application Support/wesnoth.org/iWesnoth/.wesnoth1.14/saves
Add-ons dir:     /private/var/USER/Containers/Data/Application/235F83BD-513E-413C-996C-91564162FE79/Library/Application Support/wesnoth.org/iWesnoth/.wesnoth1.14/data/add-ons
Cache dir:       /private/var/USER/Containers/Data/Application/235F83BD-513E-413C-996C-91564162FE79/Library/Application Support/wesnoth.org/iWesnoth/.wesnoth1.14/cache

Libraries
=========

Boost:     1.60
Cairo:     1.10.0 (runtime 1.10.0)
Pango:     1.28.3 (runtime 1.28.3)
SDL:       2.0.5  (runtime 2.0.5)
SDL_image: 2.0.1  (runtime 2.0.1)
SDL_mixer: 2.0.1  (runtime 2.0.1)
SDL_ttf:   2.0.14 (runtime 2.0.14)

Features
========

Experimental OpenMP support:    no
JPG screenshots:                no
Lua console completion:         no
Legacy bidirectional rendering: no
Cocoa notifications back end:   yes

**Comment by matsjoyce** [comment]
Could you list the addons you have installed, as most of those units I haven't heard of before. Google suggests that these come from Reign of the Lords, and if so it must be leaking units into LotI. Try removing it and see if the problem persists.

**Comment by kernowboy** [comment]
Well that is annoying those units have just disappeared from my recal list with all of their items, but it appears to have fixed the problem so thanks

**Comment by Dugy** [comment]
Yes, any after-maximum level advancements in LotI are specific to LotI-specific units (that is why all advancement lines end with LotI-specific units), including those added in some way, like legacies or books. Items work with other units, but there may be some quirks like ghosts able to carry armour.

Predator was level 4, no?

**Comment by kernowboy** [comment]
Nope predator was level 5

**Comment by Dugy** [comment]
Was it the LotI Predator? Because the LotI one is level 4.

**Comment by kernowboy** [comment]
Not sure, defiantly level 5. Just seem unable to attach screenshot from my iPad

**Comment by matsjoyce** [comment]
Might be more overwriting? Which other addons do you have installed? It might also be caused by Reign of the Lords, and that unit didn't self-destruct.
